Makefile

```diff
@@ -156,6 +156,14 @@ ifeq ($(COMPILER),gcc)
   OPTFLAGS := -Os -ffast-math -fno-unsafe-math-optimizations
 endif
 
+# TODO PL should be disabled for non-gamecube
+UCODE_DEFS := -DF3DEX_GBI_2 -DF3DEX_GBI_PL
```

Comment:
naming nitpicks:
* why `UCODE` instead of `GBI`? do you plan to use it for ABI too?
* `DEFS` is inconsistent with `CPP_DEFINES`, it would be nice if they were named the same (don't care which though)

Reply:
I think the only defines audio might have is for possible version differences, but it's probably better to have a different variable for those. Changed to `GBI_DEFINES`